openaddresses/be/bru/bosa-be-fr.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Suggestion to change the file name to bosa-region-brussels-fr.json etc to be more in line with openaddresses guidelines on their wiki (even if they don't seem to enforce it in the github...)

Reply:
Will rename these!